tsfile/src/main/java/org/apache/iotdb/tsfile/read/expression/IExpression.java

```diff
@@ -18,7 +18,9 @@
  */
 package org.apache.iotdb.tsfile.read.expression;
 
-public interface IExpression{
+import java.io.Serializable;
+
+public interface IExpression extends Serializable {
```

Comment:
This is a base interface +1 for this change. Just double check: Is there any concerns on this @jixuan1989 :)

Reply:
I have no more opinions.